channel-email/src/main/java/com/synopsys/integration/alert/channel/email/web/EmailGlobalConfigResponse.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * channel-email
+ *
+ * Copyright (c) 2021 Synopsys, Inc.
+ *
+ * Use subject to the terms and conditions of the Synopsys End User Software License and Maintenance Agreement. All rights reserved worldwide.
+ */
+package com.synopsys.integration.alert.channel.email.web;
+
+import com.synopsys.integration.alert.common.rest.model.Config;
+
+public class EmailGlobalConfigResponse extends Config {
```

Comment:
Because this model is used in both requests and responses, I would be in favor of using the suffix "RESTModel" or something to indicate that the object is used for both.

Reply:
Updated to Model to be consistent with the rest of Alert.

Where did Model come from? I'm seeing the term Resource a lot in RESTful documentation, but not so much Model.

Reply:
I have no preference if it is called, "model" or "resource". I just want to make it clear that the object represents more than _responses_ since it is also used for requests.